src/main/scala/com/taxis99/amazon/sqs/SqsConfig.scala

```diff
@@ -14,7 +14,7 @@ protected[this] trait SqsConfig {
     * Defines the serialization method to produce/consume messages.
     * @return The serialization object
     */
-  def serializer: ISerializer = PlayJson
+  implicit def serializer: ISerializer = PlayJson
```

Comment:
This change is a breaking change isn't it? That means you'll require to promote the minor version.